rotimi-babalola/owambe
# #145799393 - Create API endpoints

#### What does this PR do?
This PR adds a `/users` endpoint that supports CRUD operations for users. The endpoints added are as follows:

- `GET /users` : Gets all users
- `POST /users` : Creates a new user
- `GET /users/:id` : Gets a particular user 
- `PUT /users/:id` : Updates a user
- `DELETE /users/:id` : Deletes a user
- `POST /users/signin` : Sign in an existing user

#### Description of Task to be completed?
Create API endpoints that can be used for CRUD operations on users

#### How should this be manually tested?
Make sure the server is running on `localhost:3000` by running `npm run start-server`. Use Postman to hit any of the endpoints mentioned above and you should get a response

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#145799393

#### Screenshots (if appropriate)
<img width="827" alt="screen shot 2017-05-25 at 22 18 42" src="https://cloud.githubusercontent.com/assets/24937453/26471099/27ad4028-4198-11e7-85bc-77e4e8e88cb5.png">

#### Questions:
